src/client/java/minicraft/core/Renderer.java

```diff
@@ -238,7 +238,11 @@ private void initialize() {
 			HARDWARE_ACCELERATION_STATUS.initialize();
 		}
 
-		public abstract class AppStatusElement {
+		public enum GeneralStatus {
+			OFF, ON
+		}
+
+		public abstract class AppStatusElement<E extends Enum<E>> {
```

Comment:
Please remove this class. It serves no purpose.

Reply:
We don't know if this will be extended yet. Minimal implementation is fine for most uses.

Reply:
Still, this is a part of this API.

Reply:
If this will not be extended, it is whole pointless to add this API; I would rather remove this feature from this pull request.